common.go

```diff
@@ -434,3 +441,20 @@ func GetLanguageByAlias(alias string) (lang string, ok bool) {
 
 	return
 }
+
+// GetLanguagesByGitAttributes returns the language assigned in .gitattributes if the regular expresion
+// matchs with the filename
```

Comment:
Why is this function ignoring most of its arguments?

If it is to implemement and interface, it would be nice to know, and why are they being ignored in this particular implementation. Also give ignored arguments a good name, like `_`?